l5kit/l5kit/simulation/unroll.py

```diff
@@ -117,15 +117,9 @@ def unroll(self, scene_indices: List[int]) -> List[SimulationOutput]:
         """
         sim_dataset = SimulationDataset.from_dataset_indices(self.dataset, scene_indices, self.sim_cfg)
 
-        if self.sim_cfg.num_simulation_steps is None:
-            range_unroll = range(self.sim_cfg.start_frame_index, len(sim_dataset))
-        else:
-            end_frame_index = min(len(sim_dataset), self.sim_cfg.num_simulation_steps + self.sim_cfg.start_frame_index)
-            range_unroll = range(self.sim_cfg.start_frame_index, end_frame_index)
-
-        for frame_index in tqdm(range_unroll):
+        for frame_index in tqdm(range(len(sim_dataset))):
```

Comment:
I would add a config like `show_progress` in case the user wants to hide the output, `tqdm` has an option called `disable` that is often used for that.

Reply:
the question is where should we add it so that it's visible here. Is the SImulationConfig the right place?

Reply:
Yes, I would add in the `SimulationConfig`.